pcap-dlpi.c

```diff
@@ -122,6 +122,12 @@
 
 #define	MAXDLBUF	8192
 
+/*
```

Comment:
As this is only used for HP-UX, should it be defined inside an `#ifdef` for HP-UX?